### PR TITLE
chore: update Getting Started landing page

### PR DESCRIPTION
**Pull request by khorne3**
- simplified `index.md` to draw attention to contents of the section
- moved a lot of the existing content on `index.md` to `admin.md`

### CODE REVIEW

File: getting-started/admin.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+begin working on their projects.
+
+## Set up and deploy Coder
+
+To get started with Coder, you’ll need to [deploy Coder](./setup/index.md) to a
+Kubernetes cluster. We have documentation to help you
+[create a cluster](./setup/kubernetes/index.md) if needed. Once you have a
+cluster, you can [install Coder via Helm](./setup/installation.md).
+
+Alternatively, you can leverage [Coder for Docker](../setup/docker.md), which
```
Comment (jawnsy) [inline getting-started/admin.md:17]: seems a little odd that some most links refer to `./setup` and this one is using `../setup`

File: getting-started/admin.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+terminal, and enable [file sync](../cli/file-sync.md) to have their local
+directory’s tree sync with the remote file system.
+
+## Maintain and update
+
+Coder maintains a public [changelog](../changelog/index.md) and
+[release calendar](https://coder.com/release-calendar.ical) to help you stay in
+the know on features, bug fixes, security updates, and breaking changes that are
+coming. Coder releases are available on the third Wednesday of each month, and
+patch releases are published and available as needed.
```
Comment (ericpaulsen) [inline getting-started/admin.md:75]: Since the title is `Maintain and Update`, we should link to our upgrade documentation here.

**Review by bpmct — APPROVED** [review]
This is clear to me!

However, the home of docs doesn't really link to either of these, instead linking to the "Getting started" for workspaces. And "Setup" and "Admin" CTAs. How will that work with this new getting started content?

File: getting-started/admin.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+Coder has a [command-line (CLI) tool](../cli/index.md) that you and your
+developers may be interested in using to interact with Coder. The CLI is
+completely [open-sourced](https://github.com/cdr/coder-cli), and we always
+welcome contributions. Additionally, Coder has a [public API](../guides/api.md)
+that you can use to automate various tasks through code.
+
+## Connect local IDEs
+
+While Coder supports a
+[variety of IDEs in the browser](https://coder.com/docs/coder/v1.20/workspaces/editors),
```
Comment (mterhar) [inline getting-started/admin.md:61]: Why is this a full url?

**Comment by ericpaulsen** [comment]
Generally speaking, I sense that the paragraph format might not be as inviting and efficient to read. For a Getting Started guide, I think a bulleted list of items to review, configure, etc. might be easier to read through. E.g.:

> ## Set up and deploy Coder
>
> To get started with Coder, you'll need to install the necessary infrastructure         and deploy Coder via Helm. We have documentation to walk you through each step:
>
> - [Create a Kubernetes cluster](./setup/kubernetes/index.md)
> - [Install Coder via Helm](../setup/installation.md)

etc...

Thoughts? @khorne3 @bpmct @mterhar @jawnsy

File: getting-started/admin.md (hunk 3 above)
Comment (khorne3) [reply getting-started/admin.md:61]: should be shortened

**Comment by khorne3** [comment]
@bpmct I think I'll replace the Getting Started link (in the hero image) with a link to the getting started guides instead of to `workspaces/editors` -- I think this accomplishes the same thing, but with better context and instructions

@ericpaulsen That's a good point; I'll edit and we can take a look

**Comment by ericpaulsen** [comment]
Looks good! Only a couple things before we merge:

- The numbering is off with the bulleted items in the `Configure Coder` section
- The `Additional Information` section seems to be incomplete

Thanks @khorne3

**Pull request merged**